integration-tests/src/test/java/com/datastax/oss/driver/core/cql/ExecutionInfoWarningsIT.java

```diff
@@ -221,4 +234,12 @@ public void should_expose_warnings_on_execution_info() {
                     .contains("is of size")
                     .containsPattern("exceeding specified .*threshold"));
   }
+
+  private static void sleep(long duration) {
+    try {
+      Thread.currentThread().sleep(duration);
+    } catch (InterruptedException e) {
+      Thread.currentThread().interrupt();
+    }
+  }
```

Comment:
`Uninterruptibles.sleepUninterruptibly` or using `after` instead of `timeout` would also have been fine.
This also works.